File: Java/Stars.java
import java.util.*;
import java.lang.*;

public class Stars {
	public static void main(String[] args) {
		int i, j, k, n;
		String option;
		Scanner input;
		do {
			input = new Scanner(System.in);
			System.out.println("Enter the No. of Rows: ");
			n = input.nextInt();
			for (i = 0; i < n; i++) {
				for (j = 0; j <= i; j++) {
					System.out.print("  ");
				}

				for (k = n; k > 0 + i; k--) {
					System.out.print("*  ");
				}
				System.out.println();
			}
			input = new Scanner(System.in);
			System.out.println("Continu? Y/N");
			option = input.nextLine();
			do {
				if (!option.equalsIgnoreCase("y") || !option.equalsIgnoreCase("y")) {
					input = new Scanner(System.in);
					System.out.println("Continu? Y/N");
					option = input.nextLine();
				}
			} while (!option.equalsIgnoreCase("y") || !option.equalsIgnoreCase("n"));
		} while (!option.equalsIgnoreCase("n"));
	}
}

File: Kotlin/Stars.kt
package Stars

import java.lang.*
import java.util.*

fun main(args: Array<String>) {
  var option: String
  do {
    println()

    print("Enter No. of Lines: ")

    var scan = Scanner(System.`in`)
    val rows = scan.nextInt()

    println()

    for (i in 0..rows) {
      for (j in 0 until rows - i) print(" ")

      for (j in 0..i) print("* ")

      for (j in i until rows) print("  ")

      for (j in 0..i) print(" *")

      println()
    }

    for (i in 0..rows) {
      for (j in 0..i * 2) print(" ")

      for (j in i * 2..rows * 2) print("* ")

      println()
    }

    do {
      println()

      print("Go Again?(Y/N): ")
      scan = Scanner(System.`in`)
      option = scan.nextLine()

      println()

      if (askAgain(option) == 1) println("Y/N")
    
    } while (askAgain(option) == 1)

    println("_____________________________________________________")

  } while (!option.equals("n", ignoreCase = true))

  println()

  println("Thank You!")

}

fun askAgain(option: String): Int {
  if (option.equals("n", ignoreCase = true) || option.equals("y", ignoreCase = true)) return 0
  return 1
}
